compiler/src/iree/compiler/Codegen/LLVMCPU/test/tile_and_fuse.mlir

```diff
@@ -0,0 +1,34 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-llvmcpu-tile-and-fuse{tiling-level=0}))" --split-input-file %s | FileCheck %s
+
+func.func @matmul_bias_add(%arg0 : tensor<?x?xf32>, %arg1 : tensor<?x?xf32>, %arg2 : tensor<?xf32>) -> tensor<?x?xf32> {
```

Comment:
Probably need more tests here, I can take care of add more here for the pad fusion case I am looking into.

Reply:
What kinds of test you're looking for? The pass looks for TilingInterface ops and apply the same transformation. The test contains TilingInterface ops, and the transformations are tested upstream. I think this one is enough for now?

I added a early-skip test case if the tiling sizes are all zeros.